src/schema/v2/sorts/fair_sorts.ts

```diff
@@ -16,6 +16,6 @@ export const FairSorts = {
   }),
 }
 
-export type TFairSorts = keyof typeof FAIR_SORTS
+export type TFairSort = typeof FAIR_SORTS[keyof typeof FAIR_SORTS]["value"]
```

Comment:
(thought) Re `TFairSort` -- Know this code was here before, but wanted to call out that prefixing our types with `T` or interfaces with `I` is unconventional and, at least on our FE codebases, has been explicitly avoided in the past. 

(I'm fine with moving in this direction, but we should be deliberate about this stuff and try to stick to idiom unless that idiom has a legitimate technical cost -- in which case, we should fix the problem.)

Reply:
I don't really like it either TBH — the problem I've been having lately in Metaphysics is just avoiding naming the type the same thing as the enum or type. Typically I'd name the enum object `FairSortEnum` and have the type be `FairSort`. I can give this another look though.

If we're being honest: what is conventional in Metaphysics at this point? — it's a free-for-all of disorganized code. Conventions would be lovely.

Reply:
Sometimes conventions can be defined by what a thing is not. (And tha of course can be problematic, but not necessarily so.) I hear ya tho about MP. Our TypeScript layer is still pretty new, so there's room to improve and to set some conventions down if necessary, or to correct bad ones should they be observed.